### ISSUE TITLE
Upload Software developement projects

### ISSUE THREAD

**Issue by sudip-mondal-2002** [sudip-mondal-2002/HactoberFest21#12]
You can upload apps, games, websites or anything related to software dev. in this repository

**Comment by HiruniPerera** [comment]
Hello, i would love to contribute to this can i upload a website?

**Comment by sudip-mondal-2002** [comment]
Yes just fork this repository, create a new folder and upload your codes in that, then create a pull request

**Comment by Rishavgupta12345** [comment]
can i add some of my projects here of android apps development

**Comment by sudip-mondal-2002** [comment]
Yes why not, Sure you can add any projects @Rishavgupta12345

**Comment by Rishavgupta12345** [comment]
please review and merge my pull request

**Comment by sudip-mondal-2002** [comment]
Done!!

**Comment by HiruniPerera** [comment]
@sudip-mondal-2002 I have subbmitted my pull request, Sir can you please kindly merge my pull request?Would appereciate it very much.
Thank you

**Comment by Rishavgupta12345** [comment]
why hactoberfest is showing 14 days to verify the pull request .  do you have any idea ?

am I doing a spam or wrong pull request or this happens with everyone

**Comment by sudip-mondal-2002** [comment]
They reviews if people are doing spams to get swags

**Comment by sudip-mondal-2002** [comment]
@HiruniPerera I have merged it, you can find your profile in the contributors list of this repo

**Comment by Radhey100** [comment]
Please assign me I will like

**Comment by Madhavman** [comment]
i am that only who contacted me on whatsapp from nasa app challenge 
hope yo like 
this is the link -http://127.0.0.1:5501/index.html

and files-

![1](https://user-images.githubusercontent.com/91618945/135755112-ca81c0a9-8cba-44d0-911b-b29bcaaffdaf.jpeg)
![1stlogo](https://user-images.githubusercontent.com/91618945/135755114-77eb59bc-7640-434d-8d04-c4c088840362.jpeg)
![2](https://user-images.githubusercontent.com/91618945/135755115-766449c8-e226-43c6-92ea-97529cf4ff22.jpeg)
![2ndlogo](https://user-images.githubusercontent.com/91618945/135755116-6570e37b-58c6-4aa1-b0ce-e1de150d4de4.jpeg)
![3](https://user-images.githubusercontent.com/91618945/135755117-70f45e09-d4ff-4cbb-854a-e5523e58013c.jpeg)
![3dlogo](https://user-images.githubusercontent.com/91618945/135755118-c61981ed-000b-45f0-975c-8d3919340e44.jpeg)
![4](https://user-images.githubusercontent.com/91618945/135755119-0e2ce356-af08-48d8-88ab-1b8a8e51558d.jpeg)
![4th logo](https://user-images.githubusercontent.com/91618945/135755121-93c3224a-6943-4a18-a65e-a8d0aa4624bf.jpeg)
![5](https://user-images.githubusercontent.com/91618945/135755122-a767359b-e0ce-42e5-b483-6ded7a284e10.jpeg)
![5th lgo](https://user-images.githubusercontent.com/91618945/135755123-3ff12097-55b6-468a-b143-2636b85aa242.jpeg)
![6](https://user-images.githubusercontent.com/91618945/135755124-c8ede636-9e14-48b4-9bfc-f6e3f30ab898.jpeg)
![background](https://user-images.githubusercontent.com/91618945/135755125-79d9714e-f281-4daa-b39d-3e5a91e00960.png)
![box1](https://user-images.githubusercontent.com/91618945/135755127-82f09a34-22ad-4b34-8e9a-8f6da5a7a8d4.jpeg)
![bubble](https://user-images.githubusercontent.com/91618945/135755128-77fa20ef-b5af-47d1-9a4f-41494f5cc06b.png)
![covid 5](https://user-images.githubusercontent.com/91618945/135755129-7ffb08a6-1c93-483c-9069-bd71ddf7d786.jpeg)
![covid1](https://user-images.githubusercontent.com/91618945/135755130-e64b3ce2-8359-47d0-83c6-a7164ed0d51e.png)
![covid2](https://user-images.githubusercontent.com/91618945/135755132-2a970c11-ad42-4ca6-a94d-7e28ddb3e3d6.jpeg)
![covid3](https://user-images.githubusercontent.com/91618945/135755133-a8fd35be-2af4-4cf1-809a-4d9e1d6b50b6.jpeg)
![covid4](https://user-images.githubusercontent.com/91618945/135755134-85acdee9-55b2-42b2-b8cb-2d30c86f9372.jpeg)
![fight1](https://user-images.githubusercontent.com/91618945/135755135-a862216b-5693-43c9-a084-c67fcdcdb5f3.jpeg)
![hi](https://user-images.githubusercontent.com/91618945/135755136-8373af2d-d87f-4a93-ab67-446a6f9bd3f9.jpeg)
![immune](https://user-images.githubusercontent.com/91618945/135755137-21e1adfe-cf2c-4681-82ca-b7350b99022f.jpeg)
![immune2](https://user-images.githubusercontent.com/91618945/135755138-ba24e2a1-6cb7-4d39-b4e0-44e8549c38f6.jpeg)
![immunity](https://user-images.githubusercontent.com/91618945/135755139-2e20a856-f911-45e3-a6ad-1b6cf307d38d.jpeg)
![lock](https://user-images.githubusercontent.com/91618945/135755140-8f0da235-3ac9-453d-9b38-abc92a631e0f.jpeg)
![new](https://user-images.githubusercontent.com/91618945/135755142-0e37beff-6a1d-453f-b5bf-95ff1a096cf8.jpeg)
![pp1](https://user-images.githubusercontent.com/91618945/135755143-28bb099a-b3d4-485e-b494-6d54cba26fc0.jpeg)

**Comment by sudip-mondal-2002** [comment]
upload the code, also please don't spam here

**Comment by Gourav-s12** [comment]
Hello, @sudip-mondal-2002 i have submitted a leef code. 
can i submit a project now here?
will is count in hactoberfest? 
is there a limit? (sorry, i am new)

**Comment by sudip-mondal-2002** [comment]
Yes it'll count in hacktoberfest, there is no limits

**Comment by Vrushti24** [comment]
Hey @sudip-mondal-2002 I want to add flutter application code which is tic-tac-toe app so can I ?

**Comment by Obito9596** [comment]
Hello there ! please review my Pull Request and give comment on the basis of changed code.

**Comment by Obito9596** [comment]
Hello there ! please review my Pull Request and give comment on the basis of changed code , Because i am  beginner to Open source , give your suggestion that how to find  good repo and make some changes @sudip-mondal-2002 ?

**Comment by Hacker-sagar1** [comment]
Why i am not able to post a

**Comment by sudip-mondal-2002** [comment]
@Vrushti24 yes you are welcome
This repository allows all the popular languages

**Comment by sudip-mondal-2002** [comment]
@Obito9596 I have checked all the PRs it's either merged or I added comments

**Comment by harshitsuthar77731** [comment]
please assign me

**Comment by Vrushti24** [comment]
@sudip-mondal-2002  I will make PR soon

**Comment by archi15** [comment]
Hey @sudip-mondal-2002, I want to add a CLI based application 'Airport Management System' which I have coded in C++. Will I?

**Comment by sudip-mondal-2002** [comment]
@archi15 Yes sure, go on

**Comment by Madhavman** [comment]
Please merge mine

**Comment by MariamChowdhury** [comment]
Hi, I want to work on this. Can you please assign me this issue?

**Comment by Prateek-kannojia** [comment]
Hi @sudip-mondal-2002 i want to work on this . I want to add an Android project that I have created in kotlin please assign me this issue...

**Comment by sudip-mondal-2002** [comment]
@MariamChowdhury @Prateek-kannojia it's nod needed but ok

**Comment by PrajwalHD58** [comment]
Hi, @sudip-mondal-2002  sir I want to work on this. I want to add an Android project that I have created in Java using MVVM architecture please assign me this issue that will be very helpful for me.

**Comment by Arun23397** [comment]
Hii @sudip-mondal-2002 I want  to add spring and angular application which includes microservice and sign in authentication in both spring boot and angular. can you please assign this issue to me

**Comment by Garv-M** [comment]
Hello @sudip-mondal-2002 Bhaiya, i want to add a game which i made as a project for Android, but don't know how to add it. because if i clone the repo, the project opens in the Vs code and the android project is in android studio. How can i merge them ?

**Comment by Vrushti24** [comment]
Added PR #406 u can check @sudip-mondal-2002

**Comment by Obito9596** [comment]
@sudip-mondal-2002 can pls explain me, why my PR showing excluded project i don't know why? even my PR is accepted my you as a maintainer
